HW_02/main.py

```diff
@@ -0,0 +1,63 @@
+from flask import Flask
+import pandas as pd
+import random
+import string
+
+app = Flask(__name__)
+
+HTML_TEMPLATE = """
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Test Website</title>
+    <style>
+        body {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            height: 100vh;
+            margin: 0;
+        }
+        .content {
+            text-align: center;
+            font-size: 1.5em;
+        }
+    </style>
+</head>
+<body>
+    <div class="content">
+        %s
+    </div>
+</body>
+</html>
+"""
+
+
+@app.route("/count-average-value")
+def count_average_value_students():
+    file_path = "hw.csv"
+
+    df = pd.read_csv(file_path)
+    columns = [" Height(Inches)", " Weight(Pounds)"]
+    average_value = df[columns].mean().round(2)
+    content = f"<p> The average values for all students are: average height = {average_value[' Height(Inches)']}" \
+              f" inches and average weight = {average_value[' Weight(Pounds)']} pounds! <p/>"
+    return HTML_TEMPLATE % content
+
+
+@app.route("/get-random-password")
```

Comment:
Password always should has:
    * Uppercase letters (at least one)
    * Lowercase letters (at least one)
    * Digits (at least one)
    * Special symbols (at least one)
   
   Additionally try to add not just one of these chars but random count.